tests/framework/integration/cluster.go

```diff
@@ -178,8 +178,7 @@ type Cluster struct {
 	Members       []*Member
 	LastMemberNum int
 
-	mu            sync.Mutex
-	clusterClient *clientv3.Client
+	mu sync.Mutex
```

Comment:
Is the mutex still used? Can we remove it?

Reply:
It is stilled used when mutating `Members []*Member`. https://github.com/etcd-io/etcd/blob/cd9764a99fab81788d6234d3524a08c9b893c3e4/tests/framework/integration/cluster.go#L1371-L1375

Looks like I need to group it with `Members`, does it make sense to you?

Reply:
Hmm, usually the mutex declaration is above the variables it protects so it's a little unnatural. Let's keep it as it is and revisit it in future cleanup.